SPECS/nbdkit/nbdkit.spec

```diff
@@ -0,0 +1,2116 @@
+%global _hardened_build 1
+%ifarch %{kernel_arches}
+# ppc64le broken in rawhide:
+# https://bugzilla.redhat.com/show_bug.cgi?id=2006709
+# riscv64 tests fail with
+# qemu-system-riscv64: invalid accelerator kvm
+# qemu-system-riscv64: falling back to tcg
+# qemu-system-riscv64: unable to find CPU model 'host'
+# This seems to require changes in libguestfs and/or qemu to support
+# -cpu max or -cpu virt.
+# s390x builders can't run libguestfs
+%ifnarch %{power64} riscv64 s390 s390x
+%global have_libguestfs 1
+%endif
+%endif
+# We can only compile the OCaml plugin on platforms which have native
+# OCaml support (not bytecode).
+%ifarch %{ocaml_native_compiler}
+%global have_ocaml 1
+%endif
+# Architectures where we run the complete test suite including
+# the libguestfs tests.
+#
+# On all other architectures, a simpler test suite must pass.  This
+# omits any tests that run full qemu, since running qemu under TCG is
+# often broken on non-x86_64 arches.
+%global complete_test_arches x86_64
+# If the test suite is broken on a particular architecture, document
+# it as a bug and add it to this list.
+%global broken_test_arches NONE
+# If we should verify tarball signature with GPGv2.
+%global verify_tarball_signature 1
```

Comment:
We verify signatures through the signatures JSON. Please remove `verify_tarball_signature` and its uses.

Reply:
Done